node/aas-aid/tsconfig.json

```diff
@@ -23,5 +23,5 @@
         "strictNullChecks": true
     },
     "include": ["*.ts", "**/*.ts", "../src/**/*.ts", "**/AIDSchema.json"],
-    "exclude": []
+    "exclude": ["./dist/**/*"]
```

Comment:
Not having this results in this: https://stackoverflow.com/questions/42609768/typescript-error-cannot-write-file-because-it-would-overwrite-input-file

Reply:
In general it would be better to have such a change in yet another PR which can go in very quickly. Anyhow, let's keep it in this PR ;-) 
Note: I did not run into this issue..